code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -41,6 +47,16 @@
 	eyeobj.icon = 'icons/obj/abductor.dmi'
 	eyeobj.icon_state = "camera_target"
 
+/obj/machinery/computer/camera_advanced/xenobio/Destroy()
+	stored_slimes = null
```

Comment:
would be better to remove the slimes from the list and force them onto the turf, then cut the list.